gneiss/plot/_heatmap.py

```diff
@@ -73,10 +73,21 @@ def heatmap(table, tree, mdvar, highlights=None, cmap='viridis',
     edges = edges.unstack()
     edges = pd.DataFrame({'src_node': edges.index.get_level_values(1),
                           'dest_node': edges.values})
-    edges['x0'] = [pts.loc[n].x for n in edges.src_node]
-    edges['x1'] = [pts.loc[n].x for n in edges.dest_node]
-    edges['y0'] = [pts.loc[n].y for n in edges.src_node]
-    edges['y1'] = [pts.loc[n].y for n in edges.dest_node]
+
+    edge_list = []
+    for i in edges.index:
```

Comment:
This should be ok - looping through a data frame this way is just as fast.  I can send over benchmarks if necessary.